src/jobflow_remote/cli/flow.py

```diff
@@ -176,7 +199,36 @@ def delete(
 
     to_delete = [fi.flow_id for fi in flows_info]
     with loading_spinner(processing=False) as progress:
-        progress.add_task(description="Deleting...", total=None)
+        if delete_files:
+            progress.add_task(description="Deleting files...", total=None)
+            # get all jobs to delete
+            jobs_info = jc.get_jobs_info(flow_ids=to_delete)
+            # keep a dictionary of the hosts to avoid connecting multiple times
+            hosts: dict[str, BaseHost] = {}
+            for job_info in jobs_info:
+                if job_info.run_dir:
+                    if job_info.worker in hosts:
+                        host = hosts[job_info.worker]
+                    else:
+                        host = jc.project.workers[job_info.worker].get_host()
+                        hosts[job_info.worker] = host
+                        host.connect()
```

Comment:
I think that as long as it runs just from the CLI it will just be closed by paramiko itself before terminating the execution. However it would be safer to enforce it. I will add a close

Reply:
Only comment left on my side I think (you said you'd add a close but I did not see it ? if you decided not to add it in the end, ok for me too)

Reply:
Done. I forgot to push the change.